api-gen/settings.gradle.kts

```diff
@@ -1,12 +1,13 @@
 pluginManagement {
+  val kotlinVersion: String by settings
```

Comment:
little nicer to have this typed + with the delegate, plus for some reason, my local builds weren't working w/ the `extra["kotlin.version"]` syntax.